data/mysql/schema.sql

```diff
@@ -1906,16 +1906,16 @@ CREATE TABLE `ezsearch_object_word_link` (
 
 DROP TABLE IF EXISTS `ezsearch_search_phrase`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
+/*!40101 SET character_set_client = utf8mb4 */;
 CREATE TABLE `ezsearch_search_phrase` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `phrase` varchar(250) DEFAULT NULL,
   `phrase_count` int(11) DEFAULT '0',
   `result_count` int(11) DEFAULT '0',
   PRIMARY KEY (`id`),
-  UNIQUE KEY `ezsearch_search_phrase_phrase` (`phrase`),
+  UNIQUE KEY `ezsearch_search_phrase_phrase` (`phrase` (191)),
```

Comment:
In this case, given we introduce this in a new version where we don't so much care for BC.

I'd much rather make sure we use a size that works on the column.
Why: this is search phrase, if code insert longer phrase then 191 it anyway means search on if won't really work beyond that as it won't be part of index. So unless I'm missing some thing can just as well change size of column and provide upgrade scripts for that.

Or?

Reply:
> means search on if won't really work beyond that as it won't be part of index

I think you're right. It may return multiple different results, if the first 191 chars of the phrases are the same. Agree @alongosz ?

Apropos, I should provide upgrade scripts for all the changes anyway - but we should agree on the changes first :)